docs/admin/install/check-install-version.md

```diff
@@ -3,6 +3,8 @@ title: "Checking the version of your Knative components"
 linkTitle: "Checking your install version"
 weight: 20
 type: "docs"
+aliases:
```

Comment:
Quick Q: are going to separate the entire site by Admin + Developer Docs? 

I am supportive of this idea, I just want to make sure that's what we're going for considering the sub-folder this currently sits in.

Reply:
> Quick Q: are going to separate the entire site by Admin + Developer Docs?

Miro 🍝 explains it better but there would basically be multiple categories and we'll link between docs as required:

- Getting started
- Administrator guide
- Developer guide
- CLI guide
- Knative Serving
- Knative Eventing
- FAQs
- Troubleshooting
- Reference guide

This is in flux cos as I said previously I expect this to be a living document, but this I think is a good place to start. You can see details about what I have for each topic so far in the 🍝

Reply:
We seemed to have unanimous agreement in Slack that install goes into the admin guide so I think this is the right place for it.